contract/package.json

```diff
@@ -13,21 +13,24 @@
     "start": "yarn docker:make clean start-contract print-key",
     "build": "exit 0",
     "test": "ava --verbose",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint-fix": "eslint --fix '**/*.{js,jsx}'",
-    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.{js,jsx}'",
-    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.{js,jsx}'"
+    "lint": "eslint '**/*.{js,ts}'",
+    "lint-fix": "eslint --fix '**/*.{js,ts}'",
+    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.js'",
+    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.js'",
+    "typecheck": "tsc"
   },
   "devDependencies": {
-    "agoric": "^0.21.2-u12.0",
     "@agoric/deploy-script-support": "^0.10.4-u12.0",
+    "@agoric/eslint-config": "dev",
     "@endo/bundle-source": "^2.8.0",
     "@endo/eslint-plugin": "^0.5.2",
     "@endo/init": "^0.5.60",
+    "@endo/promise-kit": "^1.0.1",
```

Comment:
**important**
That version smells like trouble. We need to keep the endo versions consistent, and I'm pretty sure this dapp is not up to the 1.x stuff yet.

cf
 - https://github.com/Agoric/agoric-sdk/issues/8621
esp
 - https://github.com/agoric-labs/dapp-swaparoo/pull/5

Reply:
Good catch. Updated to `0.2.56`